generic/tests/cfg/trans_hugepage.cfg

```diff
@@ -29,7 +29,8 @@
             check_cmd_timeout = 900
         - relocated:
             type = trans_hugepage_relocated
-            thp_test_config = "/sys/kernel/mm/redhat_transparent_hugepage/khugepaged/pages_to_scan:4096;/sys/kernel/mm/redhat_transparent_hugepage/khugepaged/scan_sleep_millisecs:10000;/sys/kernel/mm/redhat_transparent_hugepage/khugepaged/alloc_sleep_millisecs:60000"
+            thp_test_config = "/sys/kernel/mm/transparent_hugepage/khugepaged/pages_to_scan:4096;/sys/kernel/mm/transparent_hugepage/khugepaged/scan_sleep_millisecs:10000;/sys/kernel/mm/transparent_hugepage/khugepaged/alloc_sleep_millisecs:60000"
```

Comment:
redhat paths are from old rhel versions, updated to the new ones

Reply:
@YongxueHong Yanan asked for handle this in the corresponding defrag PR, see https://github.com/autotest/tp-qemu/pull/3989, sorry for the amount of dependencies :grinning: